json-csharp/HowToConvertJSONToDataTableInCSharp/HowToConvertJSONToDataTableInCSharp/ConvertMethods.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Newtonsoft.Json;
+using System.Data;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using JsonSerializer = System.Text.Json.JsonSerializer;
+
+namespace HowToConvertJSONToDataTableInCSharp
+{
+    public class ConvertMethods
+    {
+        private const string _sampleJson
+                    = @"[{""FirstName"":""Conrad"",
+                          ""LastName"":""Ushie"",
+                          ""BirthYear"":1995,
+                          ""Subject"":""Physics""}]";
+
+        public static DataTable UseNewtonsoftJson(string sampleJson = _sampleJson)
+        {
+            DataTable dataTable = new();
+            if (string.IsNullOrWhiteSpace(sampleJson))
+            {
+                return dataTable;
+            }
+            dataTable = JsonConvert.DeserializeObject<DataTable>(sampleJson);
```

Comment:
```suggestion

            dataTable = JsonConvert.DeserializeObject<DataTable>(sampleJson);
```

Please add a new line before the line.